src/python/ensembl/io/genomio/integrity.py

```diff
@@ -404,6 +344,128 @@ def get_agp_seq_regions(self, agp_dict):
 
         return seqr
 
+
+class IntegrityTool:
+    """Check the integrity of sequence and annotation files in the genome"""
+
+    def __init__(self, manifest_file: Path, brc_mode: bool = False, ignore_final_stops: bool = False) -> None:
+        self.manifest = Manifest(manifest_file)
+        self.brc_mode = False
+        self.set_brc_mode(brc_mode)
+        self.ignore_final_stops = ignore_final_stops
+        self.errors: List[str] = []
+
+    def add_error(self, error_str: str) -> None:
+        """Store a new error in the list."""
+        self.errors += error_str
+
+    def add_errors(self, errors: List[str]) -> None:
+        """Store a list of errors in the list."""
+        for error in errors:
+            self.add_error(error)
```

Comment:
```suggestion
    def add_errors(self, *args: str) -> None:
        """Store the given errors in the list."""
        for error in args:
            self.errors += error
```
This is the most compact solution, and only requires to add below either the `s` when passing a string, or the `*` when passing a list in a variable.